testbed/addons/flux/construct.ts

```diff
@@ -21,10 +21,10 @@ export class Flux extends cdk.Construct {
         const fluxManifest = props.cluster.addManifest(
             'flux', ...yaml.loadAll(request.default(
                 "GET", "https://github.com/fluxcd/flux2/releases/download/v0.15.0/install.yaml").getBody().toString()));
-
-        props.repositories.forEach(function (value) {
+        
```

Comment:
Take a look at your editors "remove whitespace on save" feature to catch things like this.